heartbeat/monitors/active/http/_meta/fields.yml

```diff
@@ -13,6 +13,12 @@
           migration: true
           description: >
             Service url used by monitor.
+        - name: url.raw
```

Comment:
I'm wondering if this will actually work to have a field which was a multi field before pointing to an alias. Problem is that we have now an alias `http.url` and `http.url.raw` which I would think is not possible in a template.

If you run `make update` and try to load the template, you will definitively find out.